docs/sources/upgrading/_index.md

```diff
@@ -95,6 +95,21 @@ level=info ts=2022-12-20T15:27:54.858554127Z caller=metrics.go:147 component=fro
 
 These statistics are also displayed when using `--stats` with LogCLI.
 
+#### Multi-store support for compactor
+
+In releases prior to 2.7.4, setting `-boltdb.shipper.compactor.shared-store` configured the following:
```

Comment:
Not sure when this PR is going live/getting released, do we need to update 2.7.4 to read 2.7.5 now?

Reply:
agree, I'll update it